app/services/open_library_service.rb

```diff
@@ -0,0 +1,42 @@
+class OpenLibraryService
```

Comment:
We should also test this Service with rspec. For that purposes we need to mock the response of the external API. I recommend that you read the following article: https://8thlight.com/blog/dariusz-pasciak/2016/09/08/better-rspec-stubbing.html

We can make a call after that